app/static/sass/main.scss

```diff
@@ -3,3 +3,2921 @@
 main {
   @extend %site-width-container;
 }
+
```

Comment:
It's actually generated css, not scss.

As I understand it, we've got GOV.UK template (https://github.com/alphagov/govuk_template) with the associated assets, and GOV.UK frontend toolkit (https://github.com/alphagov/govuk_frontend_toolkit) within the app.

Has this been lifted directly from the prototype? The prototype doesn't have production level front end code in it, and it shouldn't be used as a source of code.